jdbc/src/test/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaDataTest.java

```diff
@@ -303,12 +340,17 @@ void testGetClientInfoPropertiesOfColInfo() throws SQLException {
    *
    * @throws SQLException If an error occurs while retrieving the value.
    */
-  private void initializeWithResult() throws SQLException {
+  private void initializeWithTwoResults() throws SQLException {
```

Comment:
Since `initializeWithTwoResults` is only called by getSchemas tests, code from line 357-374 is not needed, since they are for mocking tables/columns